src/main/java/myPck/database/utils/DbManager.java

```diff
@@ -65,6 +65,11 @@ public void addSampleData() {
             Service service = this.populateService(car, client);
             Invoice invoice = this.populateInvoice(dateOfInvoice[index],priceInvoice[index]);
             ServiceReport serviceReport = this.populateServiceReport(loremIpsum[0]);
+//            long id = service.getId();
```

Comment:
Jeżeli tego nie potrzebujemy, proponuje to usunac i wyrzucić puste linie przed zamykajacymi klamrami